pynautobot/core/app.py

```diff
@@ -103,6 +105,30 @@ def custom_choices(self):
         ).get()
         return custom_fields
 
+    def custom_field_choices(self):
+        """Returns custom-field-choices response from app
+
+        :Returns: Raw response from Nautobot's custom-field-choices endpoint.
+        :Raises: :py:class:`.RequestError` if called for an invalid endpoint.
+        :Example:
+
+
+        >>> custom_field_choices_list = nb.extras.custom_field_choices()
+        >>> print(custom_field_choices_list[0]['value'])
+        First option
+        >>> print(custom_field_choices_list[0]['field']['name'])
+        test_custom_field
```

Comment:
```suggestion
       >>> nb.extras.custom_field_choices()
        [
            {
                "id": "5b39ba88-e5ab-4be2-89f5-5a016473b53c",
                "display": "First option",
                "url": "http://localhost:8000/api/extras/custom-field-choices/5b39ba88-e5ab-4be2-89f5-5a016473b53c/",
                "field": {
                    "display": "Test custom field 2",
                    "id": "5b39ba88-e5ab-4be2-89f5-5a016473b53c",
                    "url": "http://localhost:8000/api/extras/custom-fields/5b39ba88-e5ab-4be2-89f5-5a016473b53c/",
                    "name": "test_custom_field_2"
                },
                "value": "First option",
                "weight": 100,
                "created": "2023-04-15",
                "last_updated": "2023-04-15T18:11:57.163237Z"
            },
        ]
```
Same here, give full example of what this method is returning.